cfgov/unprocessed/apps/owning-a-home/js/explore-rates/data-loader.js

```diff
@@ -0,0 +1,71 @@
+import config from '../../config.json';
+
+/**
+ * Make an XHR request.
+ * @param {string} method - 'GET' or 'POST'.
+ * @param {string} url - The URL to request.
+ * @param {Object} data - Data to pass to the request.
+ * @returns {Promise} A promise for the XMLHttpRequest request.
+ */
+function _makeRequest( method, url, data ) {
+
+  if ( method === 'GET' ) {
+    let queryString = '?';
+    for ( const item in data ) {
+      queryString += `${ item }=${ data[item] }&`;
+      queryString = queryString.substring( 0, queryString.length - 1 );
+    }
+    url += queryString;
+    data = null;
+  }
+
```

Comment:
I would switch to axios and then ponyfill the Promise library. That is what we are doing here: https://github.com/cfpb/rural-or-underserved-test/pull/159/files#diff-0703b3e5822e98c1ad857ac0cdcd0756

Reply:
Whoa, nice!! How'd you figure it out? Let's merge this PR and I'll open a new one with that branch.

Reply:
I just added console.log statements in jsdom.